src/fsharp/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.fs

```diff
@@ -61,11 +61,11 @@ module FSharpDependencyManager =
                     let setVersion v = Some { current with Version = v }
                     match opt with
                     | Some "include", Some v -> addInclude v |> parsePackageReferenceOption' rest implicitArgumentCount
-                    | Some "include", None -> raise (ArgumentException(sprintf "%s requires a value" "Include"))               // @@@@@@@@@@@@@@@@@@@@@@@ Globalize me please
+                    | Some "include", None -> raise (ArgumentException(sprintf "%s requires a value" "Include"))                    // @@@@@@@@@@@@@@@@@@@@@@@ Globalize me please
```

Comment:
> Globalize me please

Are you intending on plumbing through a string from a settings file given comments like this?